windows/src/README.md

```diff
@@ -144,26 +170,29 @@ You should verify the install location of Git on your computer as it may vary.
 
 ### USERDEFINES - User Defines
 
-You can specify defines that will not be added to the git repository and will be used in
-the build in the UserDefines.mak file in the root folder. This is used mostly for
-code signing certificates. If not specified, a test certificate will be used to sign
-executables when you build a release.
+You can specify defines that will not be added to the git repository and will be
+used in the build in the UserDefines.mak file in the root folder. This is used
+mostly for code signing certificates. If not specified, a test certificate will
+be used to sign executables when you build a release.
 
-To include UserDefines.mak in the build, use the command line parameter `-DUSERDEFINES`. You
-can also set an environment variable `USERDEFINES=1` to get the same result.
+To include UserDefines.mak in the build, use the command line parameter
+`-DUSERDEFINES`. You can also set an environment variable `USERDEFINES=1` to get
+the same result.
```

Comment:
We should add a note that `USERDEFINES` should only be set if there is a `UserDefines.mak` file, otherwise the build will fail.